plugin/src/test/java/io/jenkins/plugins/casc/vault/VaultTestUtil.java

```diff
@@ -42,7 +44,7 @@ public static void runCommand(VaultContainer container, final String... command)
 
     public static boolean hasDockerDaemon() {
         try {
-            return TestEnvironment.dockerApiAtLeast("1.10");
+            return !isWindows() && TestEnvironment.dockerApiAtLeast("1.10");
```

Comment:
I was running it on Windows, but does not really matter since we move the vault code outside anyway

Reply:
Well, you were running it on windows with Linux Containers.

GitHub has decided to use Windows containers on windows.
Otherwise, it would be a feature improvement to test containers detecting OSType

Reply:
I don't feel like creating a vault windows container :swea

Reply:
I don't see the fix for ci.jenkins.io? Currently your using ACI agents with no access to Docker 😓

Reply:
https://github.com/jenkins-infra/pipeline-library/pull/108 FTR. Anyway, please feel free to merge this PR

Reply:
Better to test for the existence of a `docker` executable I guess.

Reply:
No 😬 Since the vault image is only available in Linux containers. Github's windows image is running in Windows containers mode

Reply:
Okay upstreaming a better fix: https://github.com/testcontainers/testcontainers-java/pull/1780